src/_nebari/template/stages/07-kubernetes-services/modules/kubernetes/services/argo-workflows/main.tf

```diff
@@ -534,10 +538,25 @@ resource "kubernetes_manifest" "deployment_admission_controller" {
                   "value" = var.namespace
                 },
               ]
+              "volumeMounts" = [
+                {
+                  "mountPath" = "/etc/config"
```

Comment:
Is there a reason this is `/etc/config`? Could we use `/etc/argo/...` this is just a more unix(y) way.

Reply:
I'll make this update.